onnx/backend/test/runner/__init__.py

```diff
@@ -155,6 +155,36 @@ def _assert_similar_outputs(ref_outputs, outputs):  # type: (Sequence[Any], Sequ
                 rtol=1e-3,
                 atol=1e-7)
 
+    def _retry_excute(self, func, times=3, **kwargs):  # type: (Callable[..., Any], int, **Any) -> Any
+        for i in range(times):
+            try:
+                return func(**kwargs)
+            except:
```

Comment:
Do not except everything (this will catch unexpected events, e.g. user pressing ctrl-c). `except Exception` is more suitable.